man/tldr.1

```diff
@@ -37,12 +37,11 @@ render a local page for testing purposes
 0 on success, any other positive value otherwise
 .SH SEE ALSO
 The source is available at:
-.URL "https://github.com/tldr-pages/tldr-cpp-client" "" ""
+.URL "https://github.com/tldr-pages/tldr-c-client" "" ""
 .SH REPORTING BUGS
 Report bugs through the Github repository:
-.URL "https://github.com/tldr-pages/tldr-cpp-client" "" "."
+.URL "https://github.com/tldr-pages/tldr-c-client" "" "."
 .SH COPYRIGHT
 The MIT License (MIT)
 
-Copyright (c) 2016 Arvid Gerstmann.
-
+Copyright (c) 2014-2021 Arvid Gerstmann, the tldr-pages team and contributors.
```

Comment:
```suggestion
Copyright (c) 2016 Arvid Gerstmann.
Copyright (c) 2016-2021 the tldr-pages team and contributors.
```

Reply:
You think we should?  (cc @Leandros 😅)

Reply:
We really need to wait for them to reply. Perhaps someone could send an email to them?

Reply:
@sbrl 

This was shared in Matrix lately, so just wanted to drop my thoughts here.

Reference: https://matrix.to/#/!zXiOpjSkFTvtMpsenJ:gitter.im/$WRhbUxjhliYZbRx4cml-JAuysKad2QyfJvZydecTtVQ?via=gitter.im&via=matrix.org&via=one.ems.host

To the best of my knowledge, though I am not a lawyer, there is no reason we would need their permission. That's the whole point of using an open-source license to redistribute work.

It's true that we must preserve the author's copyright notice as declared in the license.

Here is a source for the sake of it:

> You should not change or remove someone else’s copyright notice unless they have expressly (in writing) permitted you to do so. This includes third parties’ notices in pre-existing code.
> 
> — https://www.linuxfoundation.org/blog/blog/copyright-notices-in-open-source-software-projects

However, it's normal and expected to add new copyright notices when new contributors make changes. If anything, this unfortunately isn't done enough.

The original PR was modifying the existing notice, so I agree that was wrong. However, appending a new line that denotes the new copyright is fine. We must not **change** or **remove** the existing copyright notice. We can ofc, **append** another copyright notice after existing ones, if non-trivial (copyrightable) changes were made and adhere to the license.

Reply:
Okay, this makes sense @SethFalco. Will give my approval. Sorry this has taken so long!